lib/tasks/migration.rake

```diff
@@ -702,9 +714,38 @@ namespace :migration do
       puts "Summary: Save file time: #{save_time}"
       puts "Summary: Add to Collection time: #{collection_time}"
       puts "Summary: Verification time: #{verify_time}"
-
+      noid_report.close
   end
 
+    def update_handle(handle,noid)
+      link = "https://era.library.ualberta.ca/files/#{noid}"
+      handle.slice!("http://hdl.handle.net/")
```

Comment:
Is it future proof/safe to have the link and handle target hard coded like this?